tests/unit/Core/Framework/Api/Controller/InfoControllerTest.php

```diff
@@ -87,7 +87,7 @@ public function testConfig(): void
         $data = json_decode($content, true);
         static::assertIsArray($data);
         static::assertArrayHasKey('version', $data);
-        static::assertSame('6.6.9999999.9999999-dev', $data['version']);
+        static::assertSame('6.6.9999999-dev', $data['version']);
```

Comment:
Is it expected that it is still 6.6? Or where it is coming from, might be that 6.6 is still referenced somewhere in the PR?

Reply:
It's mocked it think